[export] Clean up verifier [1/n].

zhxchen17 opened the pull request:
Summary: Some adjustments to verifier so that it's easier to use it correctly. We will enable verifier later, so the current diff is no-op.

Test Plan: CI

Differential Revision: D50839295




cc @avikchaudhuri @gmagogsfm @tugsbayasgalan @angelayi @suo @ydwu4

tugsbayasgalan commented on torch/_export/passes/replace_sym_size_ops_pass.py:
```diff
@@ -10,19 +9,10 @@
-                    modified = True
-        return PassResult(graph_module, modified)
+def _replace_sym_size_ops_pass(gm: torch.fx.GraphModule):
+    for module in gm.modules():
```
Why do we not do this in top level module?

zhxchen17 replied:
we do run this in the top level module.

tugsbayasgalan commented on torch/_export/verifier.py:
```diff
@@ -53,57 +49,92 @@ def _check_correct_val(val):
-class Verifier:
-    def __call__(self, gm: GraphModule) -> None:
-        self.check_valid(gm)
+class _VeriferMeta(type):
```
_VerifierMeta

zhxchen17 committed:
[export] Clean up verifier [1/n]. (#112505)

Summary:
X-link: https://github.com/pytorch/executorch/pull/1121

Pull Request resolved: https://github.com/pytorch/pytorch/pull/112505

Some adjustments to verifier so that it's easier to use it correctly. We will enable verifier later, so the current diff is no-op.

Test Plan: CI

Reviewed By: angelayi

Differential Revision: D50839295

fbshipit-source-id: 0a02d1cd409a49bb042a106b94b0fb34d9ff2960
